### ISSUE TITLE
"imsave" is too slow to check for "is_low_contrast", which is just for a warning.

### ISSUE THREAD

**Issue by cesc-park** [scikit-image/scikit-image#2531]
## Description
[Please provide a general introduction to the issue/proposal.]

For rgb (jpg) image.

```
from PIL import Image
image = Image.open(f)
image.save(save_name)

```
The code above is 10 times faster than the code below.
```
from skimage import io
image = io.imread(f)
io.imsave(save_name, image)

```
I think it is because of checking "is_low_contrast".
If this happens not only to me but also to the general, I think it is too big a waste to give a warning.

## Way to reproduce
[If reporting a bug, please include the following important information:]
- [V] Code example

**Comment by christian-rauch** [comment]
I agree with this. I am regularly exporting pixel-wise label images, where the grey value encodes the ID of an object. Obviously, these images are not natural images and have low contrast. Checking and printing this warning seems to slow down the image export by quite some factor.
I am in favour of completely removing this check. A low-contrast check is only useful for proper photo processing applications like GIMP.

**Comment by hmaarrfk** [comment]
@christian-rauch I am sympathetic with your pains.

I don't think imageio does any of these checks. skimage mostly moved to imageio in (0.15???) anyway.

**Comment by soupault** [comment]
Closed via #3653
